examples/va/nfv/ovs-dpdk-sriov/edpm/values.yaml

```diff
@@ -194,14 +194,6 @@ data:
         [compute]
         cpu_shared_set = 0-3,24-27
         cpu_dedicated_set = 8-23,32-47
-        [neutron]
-        physnets = dpdk1, dpdk2 # CHANGEME
-        [neutron_physnet_dpdk1] # CHANGEME
-        numa_nodes = 0 # CHANGEME
-        [neutron_physnet_dpdk2] # CHANGEME
-        numa_nodes = 0 # CHANGEME
-        [neutron_tunnel] # CHANGEME
-        numa_nodes = 0 # CHANGEME
```

Comment:
Why are these lines being removed?  They seem unrelated to the DT.

Reply:
reverted